packages/libopenssl/meta.yaml

```diff
@@ -21,7 +21,7 @@ build:
       ${SIDE_MODULE_CFLAGS} \
       --prefix=${WASM_LIBRARY_DIR}
 
-    sed -i 's!^CROSS_COMPILE=.*!!g' Makefile
+    ${SED} -i 's!^CROSS_COMPILE=.*!!g' Makefile
```

Comment:
Let's not use this in the recipes. I do not want package recipes to rely on some implicit alias in the makefile.